ci/taos/plugins-staging/pr-nnstreamer-ubuntu-apptest.sh

```diff
@@ -0,0 +1,142 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##
+# @file     pr-nnstreamer-ubuntu-apptest.sh
+# @brief    Check nnstreamer sample app working well.
+# @see      https://github.com/nnsuite/TAOS-CI
+# @see      https://github.com/nnsuite/nnstreamer/wiki/usage-examples-screenshots
+# @author   Sewon Oh <sewon.oh@samsung.com>
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-wait-queue
+function pr-nnstreamer-ubuntu-apptest-wait-queue(){
+    echo -e "[DEBUG] Waiting CI trigger to run nnstreamer sample app actually."
+    message="Trigger: wait queue. There are other build jobs and we need to wait.. The commit number is $input_commit."
+    cibot_pr_report $TOKEN "pending" "TAOS/pr-nnstreamer-apptest" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+}
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-ready-queue
+function pr-nnstreamer-ubuntu-apptest-ready-queue(){
+    echo -e "[DEBUG] Readying CI trigger to run nnstreamer sample app actually."
+    message="Trigger: ready queue. The commit number is $input_commit."
+    cibot_pr_report $TOKEN "pending" "TAOS/pr-nnstreamer-apptest" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+}
+
+# @brief [MODULE] TAOS/pr-nnstreamer-ubuntu-apptest-run-queue
+function pr-nnstreamer-ubuntu-apptest-run-queue() {
+    echo -e "[DEBUG] Starting CI trigger to run nnstreamer sample app actually."
+    message="Trigger: run queue. The commit number is $input_commit."
+    cibot_pr_report $TOKEN "pending" "TAOS/pr-nnstreamer-apptest" "$message" "${CISERVER}${PRJ_REPO_UPSTREAM}/ci/${dir_commit}/" "$GITHUB_WEBHOOK_API/statuses/$input_commit"
+
+    echo -e "########################################################################################"
+    echo -e "[MODULE] TAOS/pr-nnstreamer-apptest: Starting sample app test"
+    check_dependency cmake
+    check_dependency make
+    check_dependency wget
+    check_dependency python
+
+    # Set-up environment variables.
+    export NNST_ROOT="${dir_ci}/${dir_commit}/${PRJ_REPO_OWNER}"
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$NNST_ROOT/lib
+    export GST_PLUGIN_PATH=$GST_PLUGIN_PATH:$NNST_ROOT/lib
+    declare -i result=0
+
+    # Build and install nnstreamer library
+    cd ${NNST_ROOT}
+    if [[ -d ./build ]]; then
+        rm -rf ./build/*
+    else
+        mkdir build
+    fi
+    cd build
+    cmake -DCMAKE_INSTALL_PREFIX=${NNST_ROOT} \
```

Comment:
One quesiton. Can we use the 'debuild' command ? Please check it.

Reply:
As I find, For using debuild with cmake macro, we should change the nnstreamer 'debian/rules' or 'CMakeLists.txt' file.

Reply:
Time is going. So let's think about this later.